src/features/governance/components/Choices.tsx

```diff
@@ -0,0 +1,30 @@
+import { RadioButton } from 'src/components/buttons/RadioButton';
+import { VoteType } from 'src/types';
+
+export type ChoiceProps = {
+  voteType: VoteType | undefined;
+  disabled?: boolean;
+  onChange: (voteType: VoteType) => void;
+};
+
+export const Choices = ({ voteType, disabled, onChange }: ChoiceProps) => {
+  return (
+    <div className="flex flex-col bg-primary p-[8px] rounded-[16px] gap-4 w-full">
+      <span className="font-medium text-[14px] text-color-secondary">vote</span>
+      <div className="flex flex-row justify-evenly">
+        {Object.values(VoteType)
+          .filter((x) => x !== VoteType.upvote)
```

Comment:
what if VoteType just didnt include upvote

Reply:
Yeah that's prob better